bin/resources/GameIndex.yaml

```diff
@@ -24689,6 +24774,9 @@ SLPM-65257:
 SLPM-65260:
   name: "Air Land Force"
   region: "NTSC-J"
+SLPM-65261:
+  name: "TBS All-Star Kansha Matsuri Vol .1 - Chou Gouka! Quiz Ketteiban"
```

Comment:
``Kansha-sai`` is better.

Reply:
I used redump listing:
http://redump.org/disc/38164/

Ironically on psxdatabase:
https://psxdatacenter.com/psx2/games2/SLPM-65261.html

On top it's different to official naming:
![image](https://user-images.githubusercontent.com/24227051/163433814-c564e6b9-0b8f-4a2d-b8f4-d8baae7296f3.png)

Serialstation:
https://www.serialstation.com/titles/SLPM/65261

But if you look at the title Kansha Masture means Appreciation Festival and Kansha-sai is more like Thanksgiving Festival.

I can change it though.